data/json/items/comestibles/carnivore.json

```
@@ -279,8 +279,8 @@
     "copy-from": "human_flesh",
     "type": "ITEM",
     "subtypes": [ "COMESTIBLE" ],
-    "name": { "str_sp": "cooked pepper creep" },
-    "description": "A cooked slice of some unpleasant person properly seasoned with black pepper and salt.  Tastes great.",
+    "name": { "str_sp": "cooked pepper human" },
```

```suggestion
    "name": { "str_sp": "peppered person" },
```
'cooked peppered creep' is a terrible name IMO. Let's keep the theme going.